ui/app/templates/components/identity/item-details.hbs

```diff
@@ -1,17 +1,16 @@
 <div class="box is-shadowless is-marginless is-fullwidth">
   {{#if @model.disabled}}
-    <AlertBanner
-      @type="warning"
-      @title="Attention"
-      @message="This {{@model.identityType}} is disabled. All associated tokens cannot be used, but are not revoked."
-      data-test-disabled-warning
-    >
+    <Hds::Alert data-test-disabled-warning @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+      <A.Title>Attention</A.Title>
+      <A.Description>
+        This
+        {{@model.identityType}}
+        is disabled. All associated tokens cannot be used, but are not revoked.
+      </A.Description>
       {{#if @model.canEdit}}
-        <button onclick={{action "enable" @model}} type="button" class="link" data-test-enable={{true}}>
-          Enable
-        </button>
+        <A.Button data-test-enable-identity @text="Enable" @color="secondary" {{on "click" (action "enable" @model)}} />
```

Comment:
Consulted design and intentionally changed this button to look like a button (not a link)
![Screenshot 2023-06-14 at 4 04 17 PM](https://github.com/hashicorp/vault/assets/68122737/c39a436d-cf69-4595-8458-fa72e0e44dea)

[secondary](https://helios.hashicorp.design/components/alert#buttons) class is for alert action buttons